### PR TITLE
Create Login Activity

### PR DESCRIPTION
**Pull request by bazadactyl**
The authentication is working, but it's "primitive" for the time being. I'll still have to do the following before closing this PR:
- [x] clean up the code (remove the unnecessary stuff)
- [x] use strings.xml instead of ConstantValues.java to store consumer key and consumer secret
- [x] make the LoginActivity UI look nice
- [ ] store token and secret token for user in SharePreferences

### CODE REVIEW

File: app/src/main/java/com/dankideacentral/dic/ConstantValues.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.dankideacentral.dic;
+
+/**
+ * Created by basim on 10/6/16.
+ * This file was part of the tutorial I was using. 
+ *
+ * TODO: Use SharedPreferences instead of this shitty file.
+ */
+
+public class ConstantValues {
```
Comment (srowhani) [inline app/src/main/java/com/dankideacentral/dic/ConstantValues.java]: Instead of a ConstantValues class, I believe it would be considered better practice to have resources in the `strings.xml` file.

**Review by srowhani — CHANGES_REQUESTED** [review]
Looks good :+1:

File: app/src/main/java/com/dankideacentral/dic/ConstantValues.java (hunk 1 above)
Comment (bazadactyl) [reply app/src/main/java/com/dankideacentral/dic/ConstantValues.java]: I was initially thinking of making a second pull request to fix that issue, but perhaps it's better for me to commit to this feature until I'm officially done with the LoginActivity issue.

tl;dr I'll do that. Don't merge until I'm done with the items I listed at the top.

**Comment by bazadactyl** [comment]
@srowhani @zandermax @ermel272: is it better practice to group all the authentication-related class files under com.dankideacentral.dic.authentication?

**Comment by srowhani** [comment]
Good call

**Review by srowhani — APPROVED** [review]
Good stuff! Going to merge
